docs/examples/floating-pv/plot_floating_pv_cell_temperature.py

```diff
@@ -0,0 +1,229 @@
+r"""
+Calculating the cell temperature for floating PV
+================================================
+
+This example demonstrates how to calculate the cell temperature for
+floating PV systems using the PVSyst temperature model.
+
+One of the primary benefits attributed to floating photovoltaic (FPV) systems
+is their lower operating temperatures, which are expected to increase the
+operating efficiency. In general, the temperature at which a photovoltaic
+module operates is influenced by various factors including solar radiation,
+ambient temperature, wind speed and direction, and the characteristics of the
+cell and module materials, as well as the mounting structure. Both radiative
+and convective heat transfers play roles in determining the module's
+temperature.
+
+One of the most common models for calculating the PV cell temperature is the
+empirical heat loss factor model suggested by Faiman and implemented in
+PVSyst (:py:func:`~pvlib.temperature.pvsyst_cell`). The PVSyst model for cell
+temperature :math:`T_{C}` is given by
+
+.. math::
+    :label: pvsyst
+
+    T_{C} = T_{a} + \frac{\alpha \cdot E \cdot (1 - \eta_{m})}{U_{c} + U_{v} \cdot WS}
+
+Where :math:`E` is the plane-of-array irradiance, :math:`T_{a}` is the
+ambient air temperature, :math:`WS` is the wind speed, :math:`\alpha` is the
+absorbed fraction of the incident irradiance, :math:`\eta_{m}` is the
+electrical efficiency of the module, :math:`U_{c}` is the wind-idependent heat
+loss coefficient, and :math:`U_{v}` is the wind-dependent heat loss coefficient.
+
+However, the default heat loss coefficient values of this model were
+specified for land-based PV systems and are not necessarily representative
+for FPV systems.
+
+In FPV systems, variations in heat loss coefficients are considerable, not
+only due to differences in design but also because of geographic factors.
+Systems with extensive water surfaces, closely packed modules, and restricted
+airflow behind the modules generally exhibit lower heat loss coefficients
+compared to those with smaller water surfaces and better airflow behind the
+modules.
+
+For FPV systems installed over water without direct contact, the module's
+operating temperature, much like in land-based systems, is mainly influenced
+by the mounting structure (which significantly affects the U-value), wind,
```

Comment:
Both U-values? Or is this specific U_c or U_v?

Reply:
Thank you for the review @cwhanse! To my understanding, it would affect both coefficients if the panel is in contact with the water, but mainly the U_v if only its distance from the water and/or the adjacent panels is changed. This is why I wrote that it affects the U-value in general... Thoughts?

Reply:
"both U-value coefficients" works for me.